alcs-frontend/src/assets/env.js

```diff
@@ -0,0 +1,7 @@
+(function(window) {
```

Comment:
Do I get it right that we will be duplicating configuration to assets config files for deployment?